MFE/dashboard/src/view/widget/wizard/wizard.jsx

```diff
@@ -40,6 +40,28 @@ const Wizard = ({ initialValues, ...props }) => {
     setValues(value);
   };
 
+  const stepperNavigator = label => {
```

Comment:
You cannot define the steps inside the wizard component, each widget can have its own steps with an amount defined by the widget itself.